pkg/subnet/publisher.go

```diff
@@ -91,17 +90,16 @@ func (p *publisherImpl) Publish(
 	// TODO review these options
 	commit, err := wt.Commit(commitStr, &git.CommitOptions{
 		Author: &object.Signature{
-			Name:  "Avalanche-CLI",
-			Email: "info@avax.network", // this obviously would have to change
+			Name:  constants.GitRepoCommitName,
```

Comment:
Do we need to set this? Can we use the global value?

Reply:
Again, will try to get information from `git config`, and if not available, just use this one as a fallback.